code/controllers/subsystem/carbons.dm

```diff
@@ -0,0 +1,32 @@
+SUBSYSTEM_DEF(carbons)
+	name = "Carbons"
+	priority = FIRE_PRIORITY_CARBONS
+	flags = SS_KEEP_TIMING | SS_NO_INIT
+	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
+	wait = 2 SECONDS
+
+	var/list/processing = list()
+	var/list/currentrun = list()
+
+/datum/controller/subsystem/carbons/stat_entry(msg)
+	msg = "P:[length(processing)]"
+	return ..()
+
+/datum/controller/subsystem/carbons/fire(resumed = FALSE)
+	if (!resumed)
+		src.currentrun = processing.Copy()
+
+	//cache for sanic speed (lists are references anyways)
+	var/list/currentrun = src.currentrun
+	var/times_fired = src.times_fired
+	var/delta_time = wait / (1 SECONDS) // TODO: Make this actually responsive to stuff like pausing and resuming
+	while(currentrun.len)
+		var/mob/living/carbon/C = currentrun[currentrun.len]
```

Comment:
```suggestion
	while(length(currentrun))
		var/mob/living/carbon/C = currentrun[length(currentrun)]
```
let's get that time 0.00001% faster 😎

Reply:
![image](https://user-images.githubusercontent.com/75460809/216247570-048a6359-5d52-4353-95f9-033f2c823b38.png)